File: app/src/main/java/com/example/walletsmart/Models/SmartApiLast.java
package com.example.walletsmart.Models;

import java.io.Serializable;

public class SmartApiLast implements Serializable {
    private String id;

    private String created;

    public String getId() {
        return this.id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getCreated() {
        return this.created;
    }

    public void setCreated(String created) {
        this.created = created;
    }
}

File: app/src/main/java/cc/smartcash/smarthub/models/SmartApiLast.kt
package cc.smartcash.smarthub.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
data class SmartApiLast(

        @Expose
        @JsonProperty("id")
        @get:JsonProperty("id")
        @set:JsonProperty("id")
        @SerializedName("id")
        var id: String? = null,

        @Expose
        @JsonProperty("created")
        @get:JsonProperty("created")
        @set:JsonProperty("created")
        @SerializedName("created")
        var created: String? = null


) : Serializable
